sympy/logic/tests/test_boolalg.py

```diff
@@ -1341,12 +1341,27 @@ def test_relational_threeterm_simplification_patterns_numerically():
                 sublist = dict(zip(symb, values))
                 originalvalue = original.xreplace(sublist)
                 simplifiedvalue = simplified.xreplace(sublist)
-                assert originalvalue == simplifiedvalue, "Original: {}\nand"\
-                    " simplified: {}\ndo not evaluate to the same value for"\
-                    "{}".format(pattern[0], simplified, sublist)
+                assert originalvalue == simplifiedvalue, "Original: {}\nand" \
+                                                         " simplified: {}\ndo not evaluate to the same value for" \
+                                                         "{}".format(pattern[0], simplified, sublist)
 
 
 def test_issue_25451():
     x = Or(And(a, c), Eq(a, b))
     assert isinstance(x, Or)
     assert set(x.args) == {And(a, c), Eq(a, b)}
+
+
+def test_issue_26985():
+    a, b, c, d = symbols('a b c d')
+
+    # Expression before applying to_anf
+    x = Xor(c, And(a, b), And(a, c))
+    y = Xor(a, b, And(a, c))
+
+    # Applying to_anf
+    result = Xor(Xor(d, And(x, y)), And(x, y))
+    result_anf = to_anf(Xor(to_anf(Xor(d, And(x, y))), And(x, y)))
+
+    assert result_anf
+    assert result
```

Comment:
The assert statements looks like incomplete
```suggestion
    assert result_anf == d
    assert result == d
```